nilearn/datasets/atlas.py

```diff
@@ -1517,32 +1718,41 @@ def fetch_atlas_schaefer_2018(n_rois=400, yeo_networks=7, resolution_mm=1,
 
     Parameters
     ----------
-    n_rois : int, optional
-        Number of regions of interest {100, 200, 300, 400, 500, 600,
-        700, 800, 900, 1000}.
-        Default=400.
+    n_rois : {100, 200, 300, 400, 500, 600, 700, 800, 900, 1000}, optional
+        Number of regions of interest. Default=400.
 
-    yeo_networks : int, optional
-        ROI annotation according to yeo networks {7, 17}.
+    yeo_networks : {7, 17}, optional
+        ROI annotation according to yeo networks.
         Default=7.
 
-    resolution_mm : int, optional
-        Spatial resolution of atlas image in mm {1, 2}.
+    resolution_mm : {1, 2}, optional
+        Spatial resolution of atlas image in mm.
         Default=1mm.
     %(data_dir)s
-    base_url : string, optional
-        base_url of files to download (None results in default base_url).
+    base_url : :obj:`str`, optional
+        Base URL of files to download (``None`` results in
+        default ``base_url``).
     %(resume)s
     %(verbose)s
 
     Returns
     -------
-    data : sklearn.datasets.base.Bunch
+    data : :func:`sklearn.utils.Bunch`
         Dictionary-like object, contains:
 
-        - maps: 3D Nifti image, values are indices in the list of labels.
-        - labels: ROI labels including Yeo-network annotation,list of strings.
-        - description: A short description of the atlas and some references.
+            - 'maps': :obj:`str`, path to nifti file containing the
+              3D :class:`~nibabel.nifti1.Nifti1Image` (its shape is
+              ``(182, 218, 182)``). The values are consecutive integers
+              between 0 and ``n_rois`` which can be interpreted as indices
+              in the list of labels.
+            - 'labels': :class:`numpy.ndarray` of :obj:`str`, array
+              containing the ROI labels including Yeo-network annotation.
+
+                .. note::
+                    The list does not have 'Background' by default.
```

Comment:
so here as well indices into the list of labels are actually offset by 1 ?

Reply:
Good catch! Thanks @jeromedockes !
I improved the doc in 4a7560a